### PR TITLE
fix(ivy): don’t publish animation bindings as attributes

### PR DESCRIPTION
**Pull request by mhevery**
Some of the animation tests have been failing because animation gets 
triggered multiple times. The reason for this is that the compiler was 
generating static attribute bindings in addition to dynamic bindings.
This created multiple writes to the animation render which failed the 
tests.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/compiler-cli/test/compliance/r3_view_compiler_spec.ts [hunk 1]
```diff
@@ -120,4 +120,70 @@ describe('r3_view_compiler', () => {
       expectEmit(result.source, bV_call, 'Incorrect bV call');
     });
   });
+
+  describe('animations', () => {
+    it('should keep @attr but supress [@attr]', () => {
```
Comment (kara) [inline packages/compiler-cli/test/compliance/r3_view_compiler_spec.ts]:
```suggestion
    it('should keep @attr but suppress [@attr]', () => {
```

**Review by kara — APPROVED** [review]
LGTM, aside from typo

**Comment by kara** [comment]
@mhevery This change should also make this test pass:

https://github.com/angular/angular/blob/master/packages/platform-browser/animations/test/animation_renderer_spec.ts#L282

We should probably remove the fixme there too

File: packages/compiler-cli/test/compliance/r3_view_compiler_spec.ts (hunk 1 above)
Comment (mhevery) [reply packages/compiler-cli/test/compliance/r3_view_compiler_spec.ts]: fixed

**Comment by mhevery** [comment]
http://test/OCL:227554341:BASE:227554398:1546457906243:d5d1f0f9

**Comment by mhevery** [comment]
CARETAKER unrelated flake on travis. Please merge.

**Pull request closed without merging**